packages/lisk-transactions/test/utils/validation/validate_transaction.js

```diff
@@ -0,0 +1,135 @@
+/*
+ * Copyright © 2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+import fixtures from '../../../fixtures/transactions.json';
+import validateTransaction from '../../../src/utils/validation/validate_transaction';
+
+describe('#validateTransaction', () => {
+	describe('fixture', () => {
+		it('should be all valid for the fixtures', () => {
+			return fixtures
+				.filter(tx => tx.type !== 6 && tx.type !== 7)
+				.forEach(tx => {
+					const { valid, errors } = validateTransaction(tx);
+					expect(valid).to.be.true;
+					expect(errors).to.be.null;
+				});
+		});
```

Comment:
If we had a test for the broken multisigs, we could also test that type 6 and 7 from fixtures fail.